### PR TITLE
Console scripts example

### PR DESCRIPTION
**Pull request by dneise**
Example how console scripts would work.

Not too much code is needed for this. 
In order to try it out. you have to "install" digicampipe again, so that setup.py creates command line interfaces.

```
cd to your digicampipe folder
pip install -e .
cd somewhere else ... e.g. where your SST1M data is.
```

then do `digicam_baseline --help`
in case you are like me and have forgotten what options that program needs

then call for example: `digicam_baseline dark.npz SST1M01_20171030.002.fits.fz`
In my case it took ~25sec ... pretty nice.

### CODE REVIEW

File: digicampipe/scripts/baseline.py [hunk 1]
```diff
@@ -42,7 +42,7 @@ def main(baseline_file_path, files, unwanted_pixels=[]):
         pass
 
 
-if __name__ == "__main__":
+def entry():
     args = docopt(__doc__)
     args['--unwanted_pixels'] = [
         int(x) for x in args['--unwanted_pixels'].split(',') if x]
```
Comment (calispac) [inline digicampipe/scripts/baseline.py]: Can we still keep the "main" and call `entry()` in it ? that way you can still call it the "regular" way ?

File: digicampipe/scripts/dark_evaluation.py [hunk 2]
```diff
@@ -109,7 +109,7 @@ def main(
     plt.show()
 
 
-if __name__ == "__main__":
+def entry():
     args = docopt(__doc__)
     args['--unwanted_patches'] = [
         int(x) for x in args['--unwanted_patches'].split(',')]
```
Comment (calispac) [inline digicampipe/scripts/dark_evaluation.py]: Can we still keep the "main" and call `entry()` in it ? that way you can still call it the "regular" way ?

File: digicampipe/scripts/pipeline_crab.py [hunk 3]
```diff
@@ -151,7 +151,7 @@ def main(args):
             data_stream=data_stream, output_filename=args['--outfile_path'])
 
 
-if __name__ == '__main__':
+def entry():
     args = docopt(__doc__)
     print(args)
     args['--min_photon'] = int(args['--min_photon'])
```
Comment (calispac) [inline digicampipe/scripts/pipeline_crab.py]: Can we still keep the "main" and call `entry()` in it ? that way you can still call it the "regular" way ?

**Comment by dneise** [comment]
The "regular" way is in my opinion the "bad" way .. but sure .. technically this is possible. 
It is also kind of clean .. just two more lines at the bottom of each of these files.

**Pull request closed without merging**